CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 # Changelog
 
 ## v1.18.0 (2020-11-16)
-    Fix testing error caused by simultanious merge
+    Fix testing error caused by simultaneous merge
```

Comment:
Please do not change the changelog file.  Sadly this needs to match the actual Changelog in git.